### ISSUE TITLE
Error: Cannot start service app: OCI runtime create failed: invalid mount

### ISSUE THREAD

**Issue by meticoeus** [SonarSoftwareInc/customer_portal#121]
Getting error running install script.

AWS EC2 instance running Ubuntu 18.04 AMI
Operating System: Ubuntu 18.04.5 LTS
Kernel: Linux 5.4.0-1045-aws
Architecture: x86-64
Docker version 20.10.7, build f0df350
docker-compose version 1.23.2, build 1110ad01

End of build log:
```sh
Pulling redis (redis:5.0.4-alpine)...
5.0.4-alpine: Pulling from library/redis
Pulling watchtower (v2tec/watchtower:)...
latest: Pulling from v2tec/watchtower
Creating customer_portal_redis_1      ... done
Creating customer_portal_watchtower_1 ... done
Creating customer_portal_certbot_1    ...
Creating sonar-customerportal         ... error

ERROR: for sonar-customerportal  Cannot start service app: OCI runtime create failed: invalid mount {Destination:[/var/www/htmCreating customer_portal_certbot_1    ... done
ions:[rbind]}: mount destination [/var/www/html/storage] not absolute: unknown

ERROR: for app  Cannot start service app: OCI runtime create failed: invalid mount {Destination:[/var/www/html/storage] Type:bind Source:/var/lib/docker/volumes/3136c97ad1bd9c78755ffb284fa740df4389b1067d1a0d5139678b398c211c8f/_data Options:[rbind]}: mount destination [/var/www/html/storage] not absolute: unknown
ERROR: Encountered errors while bringing up the project.
```

**Comment by geneccx** [comment]
@meticoeus 

This should be fixed now with #122 . Please give it another go and let me know if this resolves the issue. Thanks!

**Comment by meticoeus** [comment]
Installing containerd.io version 1.4.4-1 fixed the issue.
